test/e2e/gui/objects_map/names.py

```diff
@@ -43,7 +43,9 @@
 """ Onboarding """
 # Before you get started Popup
 acknowledge_checkbox = {"checkable": True, "container": statusDesktop_mainWindow_overlay, "objectName": "acknowledgeCheckBox", "type": "StatusCheckBox", "visible": True}
+acknowledgeIndicator = {"container": acknowledge_checkbox, "objectName": "indicator", "type": "Rectangle", "visible": True}
```

Comment:
Do you have to specify the `Rectangle` type here? It is like that but could be "anything". I'd go with `Item` if you really have to

Reply:
ah good point, i can remove the type completely, i dont really rely on it